### DISCUSSION TITLE
Idee per iniziare meglio l'anno 2024/2025

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#745] [Idee e proposte]
Gestione supplenze di tutti i tipi : sia periodiche con data di inizio e data di fine , sia quelle giornaliere ,per quanto riguarda quelle giornaliere ,sarebbe opportuno che una volta che il supplente appone la propria firma nella classe ,gli risulti immediatamente un menù a tendina con la scelta del docente di quella specifica classe che sta sostituendo e la materia ,in modo da essere più trasparenti, anche al livello di responsabilità .

Poi mi risulta che giuaschool sia l'unica piattaforma ad utilizzare dei modelli di pagelle/verbali ed altri documenti diversi da quelli che sono i modelli ministeriali proposti

**Comment by luigirotondi** [comment]
Sono d'accordo. Ottima idea quando il docente firma la supplenza e seleziona il docente assente che sostituisce.
